WordPress/src/main/java/org/wordpress/android/ui/main/WPMainTabLayout.java

```diff
@@ -64,8 +63,8 @@ private void addTab(@DrawableRes int iconId, @StringRes int contentDescriptionId
      * unread notifications
      */
     void checkNoteBadge() {
-        showNoteBadge(SimperiumUtils.hasUnreadNotes());
-    }
+        showNoteBadge(false);
```

Comment:
is there any way we can determine if a user has unread notes here? rather than passing `false`